src/ResourceState.cpp

```diff
@@ -467,16 +468,17 @@ namespace D3D12TranslationLayer
             assert((Flags & SubresourceTransitionFlags::NoBindingTransitions) == SubresourceTransitionFlags::None ||
                    !bNeedsTransitionToBindState);
 
-            if (IsD3D12WriteState(after, Flags) && TransitionableResource.m_bTriggersDeferredWaits)
+            if (IsD3D12WriteState(after, Flags) && TransitionableResource.m_bTriggersSwapchainDeferredWaits)
             {
-                m_bApplyDeferredWaits = true;
+                m_bApplySwapchainDeferredWaits;

```

Comment:
This should be `m_bApplySwapchainDeferredWaits = true;` right?

Reply:
Yep, good spot!